src/Application/Kernel.php

```diff
@@ -9,46 +9,51 @@
 use ReflectionClass;
 use SplFileInfo;
 use Tempest\AppConfig;
-use Tempest\Application\Exceptions\KernelException;
+use Tempest\Bootstraps\ConfigBootstrap;
+use Tempest\Bootstraps\DiscoveryBootstrap;
 use Tempest\Container\Container;
 use Tempest\Container\GenericContainer;
 use Tempest\Database\PDOInitializer;
 use Tempest\Discovery\Discovery;
-use Tempest\Discovery\DiscoveryLocation;
 use Tempest\Http\RequestInitializer;
 use Tempest\Http\RouteBindingInitializer;
-use function Tempest\path;
 use Throwable;
 
 final readonly class Kernel
 {
     public function __construct(
-        private string $root,
+        public string $root,
         private AppConfig $appConfig,
     ) {
     }
 
     public function init(): Container
     {
-        $container = $this->initContainer();
+        $container = $this->createContainer();
 
-        $this->initDiscoveryLocations();
+        $bootstraps = [
```

Comment:
There's no need to make bootstrappers configurable from the outside for now. Let's keep it internal until we have an actualy need for making them configurable.